### PR TITLE
Wrecked Ship Farms

### PR DESCRIPTION
**Pull request by blkerby**

### CODE REVIEW

File: region/wreckedship/main.json [hunk 1]
```diff
@@ -2113,6 +2159,47 @@
...
+                    {"refill": ["Energy", "Missile", "Super"]}
+                  ]
+                },
+                {
+                  "name": "Atomics and Coverns Farm (Power Off)",
+                  "notable": false,
+                  "requires": [
+                    {"not": "f_DefeatedPhantoon"},
+                    {"or": [
+                      "Plasma",
```
Comment (osse101) [inline region/wreckedship/main.json:2188]: Spazer works really well.
Charge is a little slow maybe adding ice makes kill times better?
Morph bombs is good

I use this farm often with just power beam but you need to do moving shots.  I'm a little interested in adding a canMovingShot tech.

Speed echoes are a cool way to farm.  Horizontal spark into the mid height ones.  The ceiling drops you need to `momentumConservingTurnaround` to clip slightly into the glass off center to get the drops, but its not required to get those drops for this.

File: region/wreckedship/main.json [hunk 2]
```diff
@@ -1064,6 +1064,29 @@
                   ]
                 }
               ]
+            },
+            {
+              "id": 9,
```
Comment (osse101) [inline region/wreckedship/main.json:1069]: There could be a powered off farm at 7.  Bombs or spazer or wave.  Plasma isn't obvious how to do it.  Charge is less obvious than plasma.

File: region/wreckedship/main.json [hunk 3]
```diff
@@ -3508,6 +3616,27 @@
...
+            },
+            {
+              "id": 2,
+              "strats": [
+                {
+                  "name": "Bull Farm",
+                  "notable": false,
+                  "requires": [
+                    {"or": [
+                      "ScrewAttack",
```
Comment (osse101) [inline region/wreckedship/main.json:3628]:
```suggestion
                      "ScrewAttack",
                      "canPseudoScrew",
```

File: region/wreckedship/main.json [hunk 4]
```diff
@@ -3344,6 +3431,27 @@
...
           "to": [
+            {
+              "id": 1,
+              "strats": [
+                {
+                  "name": "Bull Farm",
+                  "notable": false,
+                  "requires": [
+                    {"or": [
+                      "ScrewAttack",
```
Comment (osse101) [inline region/wreckedship/main.json:3442]:
```suggestion
                      "ScrewAttack",
                      "canPseudoScrew",
```

File: region/wreckedship/main.json (hunk 1 above)
Comment (blkerby) [reply region/wreckedship/main.json:2188]: Charge seems too slow, but ice alone seems fine.

Comment (osse101) [reply region/wreckedship/main.json:2208]: I just assumed ice alone would work like power beam 🤔

**Pull request merged**